frontend/src/components/indices/IndicesValidationResult.js

```diff
@@ -0,0 +1,196 @@
+import React, {useEffect, useState} from "react";
+import moment from "moment";
+import {connect} from "react-redux";
+import {useHistory, useParams, useLocation} from "react-router-dom";
+import {
+  Descriptions,
+  List,
+  Collapse,
+  Tag,
+  Table,
+  Typography,
+  Alert,
+  Tooltip,
+  Button
+} from "antd";
+const { Title, Text } = Typography;
+const { Panel } = Collapse;
+
+import AppPageHeader from "../AppPageHeader";
+import PageContent from "../PageContent";
+import api, {withToken} from "../../utils/api";
+import {list} from "../../modules/indices/actions";
+
+
+const mapStateToProps = state => ({
+  token: state.auth.tokens.access,
+  indicesTotalCount: state.indices.totalCount,
+  indicesByID: state.indices.itemsByID,
+  indices: state.indices.items,
+  isFetching: state.indices.isFetching,
+});
+
+const actionCreators = {list};
+
+const IndicesValidationResult = ({token, indicesTotalCount, indicesByID, indices, isFetching, list}) => {
+  const history = useHistory();
+  const { state } = useLocation();
+
+  const {results, validation_errors, warnings} = state.response
+  const indicesValidated = results.header
+  const allIndicesLoaded = indicesValidated?.every(index => index in indicesByID)
+  const collisions = []
+
+  //TODO: get it from Redux Store
+  if (!allIndicesLoaded)
+    list({"id__in":indicesValidated.join()})
```

Comment:
Very close to List could be clearer and call it listIndices.

Reply:
It is List